### ISSUE TITLE
Please publish for Scala 2.12.19 and 2.13.13

### ISSUE THREAD

**Issue by SethTisue** [typelevel/kind-projector#270]
...which is now on Maven Central, as per https://contributors.scala-lang.org/t/scala-2-12-19-release-planning/6216/13

**Comment by SethTisue** [comment]
broadened to include 2.13.13, which is now on Maven Central, as per https://contributors.scala-lang.org/t/scala-2-13-13-release-planning/6315/17?u=sethtisue

**Comment by SethTisue** [comment]
this probably auto-closed because 2.12.19 is done, but is 2.13.13 done...?

**Comment by armanbilge** [comment]
No, 2.13 is not done. Sorry, I didn't realize you doubled-up the issue 😅

**Comment by SethTisue** [comment]
@armanbilge 2.13.13 soon...?

**Comment by armanbilge** [comment]
https://repo1.maven.org/maven2/org/typelevel/kind-projector_2.13.13/0.13.3/kind-projector_2.13.13-0.13.3.pom
